From d3a830280f647a9cd1adbc2cf5fa6cbef0ab17f6 Mon Sep 17 00:00:00 2001
From: Fabian Kosmale <fabian.kosmale@qt.io>
Date: Tue, 26 Aug 2025 11:53:59 +0200
Subject: [PATCH 049/239] CRA review: Mark qmlintegration as insignificant

The header is only used to add meta-data to source files.

Pick-to: 6.9 6.8
Fixes: QTBUG-136199
Change-Id: Ib2a5c55b68613b9d2c3d7bd619841358201b9341
Reviewed-by: Ulf Hermann <ulf.hermann@qt.io>
(cherry picked from commit 4bab3e2c7ad61df97aa962d2b513bddaca3a3798)
Reviewed-by: Qt Cherry-pick Bot <cherrypick_bot@qt-project.org>
---
 src/qmlintegration/qqmlintegration.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/qmlintegration/qqmlintegration.h b/src/qmlintegration/qqmlintegration.h
index 0cea1bab30..9fc42b6da1 100644
--- a/src/qmlintegration/qqmlintegration.h
+++ b/src/qmlintegration/qqmlintegration.h
@@ -1,5 +1,6 @@
 // Copyright (C) 2021 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR LGPL-3.0-only OR GPL-2.0-only OR GPL-3.0-only
+// Qt-Security score:insignificant
 
 #ifndef QMLINTEGRATION_H
 #define QMLINTEGRATION_H
-- 
2.51.2

